tests/prov/test_prov_pods_deployment.py

```diff
@@ -224,26 +224,26 @@ def test_40373(self):
         """
         Test to verify the data-only pod being deployed
         """
-        config_list = self.deploy_lc_obj.get_durability_config(num_nodes=
-                                                               len(self.worker_node_list))
+        config_list = self.deploy_obj.get_durability_config(num_nodes=
+                                                            len(self.worker_node_list))
         config = secrets.choice(config_list)
         self.log.info("config is picked :%s", config)
         self.log.info("Running %s N with config %s+%s+%s", (self.num_nodes - 1),
                       config['sns_data'], config['sns_parity'],
                       config['sns_spare'])
-        self.deploy_lc_obj.test_deployment(sns_data=config['sns_data'],
-                                           sns_parity=config['sns_parity'],
-                                           sns_spare=config['sns_spare'],
-                                           dix_data=config['dix_data'],
-                                           dix_parity=config['dix_parity'],
-                                           dix_spare=config['dix_spare'],
-                                           cvg_count=config['cvg_count'],
-                                           data_disk_per_cvg=config['data_disk_per_cvg'],
-                                           master_node_list=self.master_node_list,
-                                           worker_node_list=self.worker_node_list,
-                                           deployment_type=self.prov_cfg["deployment_type_data"],
-                                           client_instances=self.prov_cfg["data_client_instance"],
-                                           destroy_setup_flag=False)
+        self.deploy_obj.test_deployment(sns_data=config['sns_data'],
```

Comment:
Looking at the tests they seem to be a good candidate for data driven tests which could reduce the complexity and make them more streamlined.